server/models/index.js

```diff
@@ -1 +1,76 @@
-index.js
\ No newline at end of file
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+mongoose.connect('mongodb://localhost/hunch');
+
+
+const UserSchema = new Schema({
+    name: {
+        type: String
+    },
+    points: {
+        type: Number
+    },
+    answer: { 
+        type: Schema.Types.ObjectId,
+        ref: 'Answer'
+    }
+})
+
+const TopicSchema = new Schema({
+    topic: {
+        type: String
+    }
+})
+
+const GameSchema = new Schema({
+    currentTopic: { 
+        type: Schema.Types.ObjectId,
+        ref: 'Topic' },
+    round: {
+        type: Number
+    },
+    users: [UserSchema],
+    topics: [TopicSchema]
+})
+
+const SelectionSchema = new Schema({
+    guessing_user: {
```

Comment:
I'm a bit confused about this schema and the answers schema.
it's complicated so I'm trying to figure out if this is correct but I'm not sure.

either way, this is still only going to be used as a sort of sub-schema for the answers schema? (same issue as above, referencing the schema vs referencing the model) if so, we don't need to turn it into a model on line 66